app/src/main/java/com/framgia/wsm/data/source/UserRepository.java

```diff
@@ -23,7 +23,18 @@ public UserRepository(UserLocalDataSource localDataSource,
     }
 
     public Observable<BaseResponse<SignInDataResponse>> login(String userName, String passWord) {
-        return mRemoteDataSource.login(userName, passWord);
+        //todo edit later
+        User user = new User();
+        user.setName("Nguyễn Văn A");
+        user.setCode("B1228368");
+        user.setEmail("nguyen.van.a@framgia.com");
+        SignInDataResponse signInDataResponse = new SignInDataResponse();
+        signInDataResponse.setUser(user);
+        signInDataResponse.setAuthenToken("12345678");
+        BaseResponse<SignInDataResponse> baseResponse =
+                new BaseResponse<SignInDataResponse>(signInDataResponse) {
+                };
```

Comment:
ERROR: 'object def rcurly' have incorrect indentation level 16, expected level should be one of the following: 8, 12.